descriptors/ModuleDescriptor-template.json

```diff
@@ -312,6 +312,37 @@
         }
       ]
     },
+    {
+      "id": "finance-storage.ledger-rollovers-budgets",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance-storage/ledger-rollovers-budgets",
+          "permissionsRequired": ["finance-storage.ledger-rollovers-budgets.collection.get"]
+        },
+        {
+          "methods": ["POST"],
```

Comment:
Regarding technical design documentation only 2 endpoints necessaary:
![image](https://user-images.githubusercontent.com/25097693/183359738-75b61c1c-d132-4e00-830b-0ca4a38fc0eb.png)
In order to avoid generation POST, PUT, DELETE by raml we need to configure only GETs in the corresponding raml file

Reply:
Thank you. Decreased count of endpoint methods